content/en/tracing/trace_collection/otel_instrumentation/ruby.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Custom Instrumentation of Ruby Applications with the OpenTelemetry API
+kind: documentation
+description: 'Instrument your Ruby application with OTel API to send traces to Datadog.'
+code_lang: ruby
+type: multi-code-lang
+code_lang_weight: 20
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
```

Comment:
Just noting that we have further reading links in the front matter, but we don't have the Further Reading section at the bottom of the article.